packages/runtime/test-runtime-utils/src/mocksForReconnection.ts

```diff
@@ -121,6 +121,21 @@ export class MockContainerRuntimeFactoryForReconnection extends MockContainerRun
 		return containerRuntime;
 	}
 
+	public getMinSeq(): number {
+		// We overwrite the min sequence number function, to simulate that
+		// the minimum sequence number will be incremented by the server while the
+		// client is not connected. By checking, whether a client is currently
+		// within the quorum we ignore it during the min sequence number computation
+		// it it is currently not connected.
```

Comment:
I find this phrasing confusing for a few reasons:

-  I think "the minimum sequence number will be incremented by the server while the client is not connected" is missing a "not": this case will NOT increment, correct?
- Also, generally our comments don't use pronouns. No need to say "we" here. The fact that this is referring to the code in this method can be assumed.
- What does "overwrite" mean here? Maybe its implying [override](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html#override-and-the---noimplicitoverride-flag) but this does not seem to be an override (if it is, please mark it as such).

With no documentation on this function, its unclear to me what exactly this function is supposed to return? 

Is this supposed to be the minimum reference sequence number any future op from any future remote edit might have as its reference sequence number? If so, I'm not sure this implementation is actually correct. When a new client joints the quorum, do we actually guarantee its sequence number is greater or equal to all existing quorum members? Also if the quorum is empty, the min sequence number should be lowered to 0, since it should only increase monotonically. I don't understand this code well enough to know exactly what the right thing to do is here, but this seems like it might have issues or at least needs more clear documentation.

Reply:
@CraigMacomber As far as I understood mock server, client, deltaManager etc., I'll try to answer while Roland is not back yet.

You can find my suggested corrections in https://github.com/ruiterr/FluidFramework/pull/1.

> I find this phrasing confusing for a few reasons...

Done. It is actually an override. I am adding `noImplicitOverride: true` flag and changing the code in this package accordingly.

> Is this supposed to be the minimum reference sequence number any future op from any future remote edit might have as its reference sequence number?

Yes, but only when processing the messages from connected clients, so that

> When a new client joints the quorum, do we actually guarantee its sequence number is greater or equal to all existing quorum members?

no, but it is guaranteed for client's messages while processing and the client is connected. If the client is not connected, the messages are stored as "pending" and no processing takes place, so `minimumSequenceNumber` is irrelevant in that case. Same is true for

> Also if the quorum is empty, the min sequence number would be lowered to 0, but this seems wrong since I think it should only increase monotonically.

Again, since the sequence numbers advance only when the messages are processed, and only for connected clients, that's not a problem.

If one looked at this function in the context of the mock server/client and message processing, it becomes clear, what it does. So, I am adding a documentation according to that and also based on other `minimumSequenceNumber` documentations found in the framework (I think, they were written with respect to the usage context as well).

Reply:
@andre4i - Could you have a look at the changes to this file?